purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -743,12 +745,17 @@ internal class BillingWrapper(
      * This prevents ANRs, see https://github.com/android/play-billing-samples/issues/310
      */
     private fun retryBillingServiceConnectionWithExponentialBackoff() {
```

Comment:
this is not really exponential anymore since it's only one retry?

Reply:
Well, it's exponential, but you can only keep on retry scheduled at the same time. So only after the currently scheduled retry finishes, will there be another retry with a longer backoff. In theory, we should only connect once, so I think it's ok?

Reply:
Ahh that's an important detail, right. I misunderstood what this was doing, that's kinda why I asked lol. Makes sense yes.